src/Sentry/buildTransitive/Sentry.targets

```diff
@@ -94,10 +94,20 @@
     </PropertyGroup>
   </Target>
 
-  <!-- Upload symbols (and any other debug information files) to Sentry after the build. -->
-  <Target Name="UploadSymbolsToSentry" AfterTargets="AfterBuild" DependsOnTargets="PrepareSentryCLI" Condition="'$(SentryUploadSymbols)' == 'true' And '$(SentryCLI)' != ''">
-    <Message Importance="High" Text="Preparing to upload debug symbols to Sentry for $(MSBuildProjectName) ($(Configuration)/$(TargetFramework))" />
-    <Exec Command="&quot;$(SentryCLI)&quot; upload-dif $(SentryCLIOptions) $(IntermediateOutputPath)" />
+  <!-- Upload debug information files to Sentry after the build. -->
+  <Target Name="UploadSymbolsToSentry" AfterTargets="Build;Publish" DependsOnTargets="PrepareSentryCLI" Condition="'$(SentryUploadSymbols)' == 'true' And '$(SentryCLI)' != ''">
+
+    <Message Importance="High" Condition="'$(SentryUploadSources)' != 'true'"
+      Text="Preparing to upload debug symbols to Sentry for $(MSBuildProjectName) ($(Configuration)/$(TargetFramework))" />
+
+    <Message Importance="High" Condition="'$(SentryUploadSources)' == 'true'"
```

Comment:
Should we avoid this if `EmbedAllSources` is true? Since we already have sources in PDB?

Reply:
it won't hurt to leave it there. CLI checks whether the uploaded file embeds sources and skips it in that case

Reply:
nice! but might as well, just to give a better message.

Reply:
Done.